doc/release/upcoming_changes/16056.deprecation.rst

```diff
@@ -0,0 +1,13 @@
+Inexact matches for mode, order and searchside are deprecated
+-------------------------------------------------------------
+Inexact and case insensitive matches for mode, order and searchside were
```

Comment:
```suggestion
Inexact matches for mode and searchside are deprecated
------------------------------------------------------
Inexact and case insensitive matches for mode and searchside were
```

Thanks for persisting @anirudh2290. I will do a last pass today and then put this in I think!

Reply:
Thanks Sebastian!  Good catch, I have addressed your comments.